bpf-gpl/tc.c

```diff
@@ -896,7 +896,10 @@ static CALI_BPF_INLINE struct fwd calico_tc_skb_accepted(struct __sk_buff *skb,
 		// If we get here, we've passed policy.
 
 		if (nat_dest == NULL) {
-			conntrack_create(&ct_nat_ctx, CT_CREATE_NORMAL);
+			if (conntrack_create(&ct_nat_ctx, CT_CREATE_NORMAL)) {
+				CALI_DEBUG("Creating normal conntrack failed\n");
```

Comment:
Change to deny here; is that right if policy has accepted and we're creating a normal entry (so can't hit the new source filtering logic)?

Reply:
Hmmm yeah we might let this through, subsequent packets may be luckier and create an entry. If not, it is just a small perf issue in this direction. However, what if policy does not allow untracked packets back? It is as bad as dropping now, isn't it? I can revert this for sure.

Reply:
I will keep it this way for now